### DISCUSSION TITLE
Error running distributed CORE/EMANE - Chain b.1.1 already exists.

### DISCUSSION THREAD

**Discussion by derkin123** [coreemu/core#711] [Q&A] [answered]
I have been able to set up 7.3.0 CORE/ 1.2.5 EMANE on a single machine using 3 virtual nodes in a wlan node and 2 in an emane node where I can ping between the nodes in their respective network. I'm attempting to convert this to a distributed emulation, to do this I am making my emane node hosted on my distributed server as a first step. I currently have a master server with ip 10.10.17.95 and distributed with ip 10.10.17.102, both of which are ubuntu 18.04 virtualbox vms using a bridged adapter network type. 

I've configured the following as advised:
On the master:

/etc/core/core.conf
```
distributed_address = 10.10.17.95 (the master's ip)
controlnet = 10.10.17.0/24
emane_event_generate = True
```

~/.core/servers.conf
```core1 10.10.17.102 4038 (the distributed ip)```

/etc/fabric.yml:
`connect_kwargs: {"key_filename": "/home/<shared username>/.ssh/core"}`

Before I add any nodes, I have the distributed server added to the session under the name core1 and host 10.10.17.102 and have my emane node configured to run with its server config setting set to the core1 server. 

When I run everything, I get the following error about how Chain b.1.1 already exists.
![image](https://user-images.githubusercontent.com/100057696/199322425-6b30719a-9382-4ea6-8c93-d214486a9ef5.png)
![image](https://user-images.githubusercontent.com/100057696/199322287-5b3f70f6-304a-487c-9527-299d8c90a5ff.png)

b.1.1 exists in the results of ifconfig for both the master and the distributed server before running core-cleanup on either of them. When I run core-cleanup on both the master and distributed neither has this item but then running the core-daemon again still yields this error. It fields almost like its trying to perform this action twice but I'm not sure where this would be caused.

**Comment by bharnden** [comment, marked as answer]
Several things, none that will potentially solve your specific issue at the moment.

1. You mention WLAN, but that is not supported when doing distributed, which may be your current issue
2. I would only use the latest version of CORE as there are no plans to update older versions
3. ebtables is no longer used for CORE in the current release

The next release is very close, which has some bigger changes in place, in the big picture, may be worth trying that. Would help find any problems, if they still persist, before release.

**Reply by derkin123** [reply]
Removing WLAN nodes seemed to fix the problem for me, thanks for the help! I'm only using old core/emane versions to proof of concept a distributed network that is compatible with yet to be updated code.

**Reply by derkin123** [reply]
One thing I was curious about was what is needed for a distributed core install/ how do you only install what is needed for a distributed core install? I currently am using a full core install and I want to be able to reduce the install size needed on the distributed servers.

**Reply by bharnden** [reply]
On the release page there has been a distributed package to help with this case.
